rfc/rfc012-nuts-organization-credential.md

```diff
@@ -0,0 +1,118 @@
+
+## 1. Introduction
+
+Users that want to interact with another system want to use real-world names when they search for organizations. DIDs do not solve this problem.
+VCs can add this information (claims) to a DID. This RFC builds upon [RFC011](rfc011-verifiable-credential.md). 
+
+## 2. Terminology
+
+* **DID**: [Decentralized Identifiers](https://www.w3.org/TR/did-core/).
+* **VC**: [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+## 3 CredentialSubject
+
+The `credentialSubject` field contains the following:
+
+```json
+{
+    "id": "did:nuts:8jx6GU9JipE6TXY2nak8RgFXMk3zaoPWsCb53N1Zjw9R",
+    "organization": {
+        "name": "De Nootjes",
+        "city": "Eibergen"
+    }
+}
+```
+
+and has the following requirements:
+
+* all fields are required.
+* all fields are encoded as strings.  
+* **id** MUST refer to a known Nuts DID.
+
+## 4 Issuance & distribution
+
+A Nuts Organization Credential is public and MUST be published over the Nuts network. Every DID MAY issue the credential.
+The VC does not have any other requirements nor does it add requirements to other VCs.
+
+## 5 Supported proofs
+
+Only default proofs from [RFC011](rfc011-verifiable-credential.md) are supported.
+
+## 6 Trust
+
+The Nuts Organization Credential MUST be trusted manually. The mechanism for this is up to the implementation.
+
+## 7 Revocation
+
+The Nuts Organization Credential uses the default revocation mechanism as stated by [RFC011](rfc011-verifiable-credential.md).
+
+## 8 Use cases
+
+The Nuts Organization Credential MAY be used as credential in the Nuts OAuth flow as stated in [RFC003](rfc003-oauth2-authorization.md).
+The credential MAY also be used as a way to find the correct DID and its services.
+
+## 9 Privacy considerations
+
+All information in the credential is public knowledge.
+
+## 10 Services
+
+No additional services than the Nuts network are required.
+
+## 11. Example
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://nuts.nl/credentials/v1"
+  ],
+  "id": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467",
+  "type": ["VerifiableCredential", "NutsOrganizationCredential"],
+  "issuer": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY",
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "expirationDate": "2010-02-01T19:73:24Z",
+  "credentialSubject": {
+    "id": "did:nuts:8jx6GU9JipE6TXY2nak8RgFXMk3zaoPWsCb53N1Zjw9R",
+    "organization": {
+      "name": "De Nootjes",
+      "city": "Eibergen"
+    }
+  },
+  "proof": {
+    "type": "EcdsaSecp256r1Signature2019",
+    "created": "2017-06-18T21:19:10Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467#qjHYrzaJjpEstmDATng4-cGmR4t-_V3ipbDVYZrVe4A",
+    "jws": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..TCYt5X
```

Comment:
JWS is formatted different from the example in RFC0011 (this one has base64-like linebreaks) is it intentional, correct and intended?

Reply:
copy paste error from whatever interface it came from.